contracts/tokens/SecurityToken.sol

```diff
@@ -64,7 +64,12 @@ contract SecurityToken is StandardToken, DetailedERC20, ReentrancyGuard, Registr
 
     struct ModuleData {
         bytes32 name;
-        address moduleAddress;
+        address module;
+        address moduleFactory;
```

Comment:
It's a good question. I was thinking we might want to look up fees / version etc. direct from the factory rather than storing locally in the ST. Didn't seem to do much harm to add this for now incase needed in the future.